librz/core/disasm.c

```diff
@@ -303,6 +303,8 @@ typedef struct {
 	int asm_types;
 } RzDisasmState;
 
+static RzList *disasm_texts;
```

Comment:
Please don't use globals, it will cause huge issues when using in multithreaded  environment